### PR TITLE
[AssetMapper] Upgrade importmap polyfill

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? |  no
| Issues        | Fix #
| License       | MIT

Not sure if we should backport this on 6.4 ?

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]
That's for 6.4 to me: infra-related stuff are bugfixes.

**Comment by nicolas-grekas** [comment]
Thank you @smnandre.

File: src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php [hunk 1]
```diff
@@ -27,7 +27,9 @@
  */
 class ImportMapRenderer
 {
-    private const DEFAULT_ES_MODULE_SHIMS_POLYFILL_URL = 'https://ga.jspm.io/npm:es-module-shims@1.8.0/dist/es-module-shims.js';
+    // https://generator.jspm.io/#S2NnYGAIzSvJLMlJTWEAAMYOgCAOAA
+    private const DEFAULT_ES_MODULE_SHIMS_POLYFILL_URL = 'https://ga.jspm.io/npm:es-module-shims@1.10.0/dist/es-module-shims.js';
+    private const DEFAULT_ES_MODULE_SHIMS_POLYFILL_INTEGRITY = 'sha384-ie1x72Xck445i0j4SlNJ5W5iGeL3Dpa0zD48MZopgWsjNB/lt60SuG1iduZGNnJn';
```
Comment (Kocal) [inline src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php:32]: The constant is not used anywhere, is it on purpose? Maybe a comment would have been enough?

Comment (nicolas-grekas) [reply src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php:32]: it's used on 7.1 so at least it's there for the merge up

Comment (Kocal) [reply src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php:32]: Makes sense, thanks :)

**Pull request merged**